fluent/fluent.go

```diff
@@ -194,7 +205,11 @@ func (f *Fluent) close() (err error) {
 
 // connect establishes a new connection using the specified transport.
 func (f *Fluent) connect() (err error) {
-	f.conn, err = net.DialTimeout("tcp", f.Config.FluentHost+":"+strconv.Itoa(f.Config.FluentPort), f.Config.Timeout)
+	if strings.HasPrefix(f.Config.FluentNetwork, "unix") {
```

Comment:
> In net.dial, can select "unix", "unixgram" and "unixpacket".
But,
Because I do not know the support status of fluentd,

I see, thanks.

Nit:
BTW, "I don't know" has a snappish tone.
It would be better to use "I'm not sure".